src/library_fs.js

```diff
@@ -895,7 +895,7 @@ FS.staticInit();
       }
 #endif
     },
-    readlink(path) {
+    readlink(path, opts = {}) {
```

Comment:
Do we need the old behaviour?  i.e. Are there other callsites beside `__syscall_readlinkat`?

Reply:
Just that and `test_fs_readlink` which would need to be updated. It's also documented public API but the documentation was wrong before and this updates the behavior to match what the documentation says.

Reply:
Actually `test_fs_readlink` doesn't cover relative links.